CHANGELOG.md

```diff
@@ -1,10 +1,10 @@
 Yii Framework 2 authclient extension Change Log
 ===============================================
 
-2.2.15 under development
+2.2.15 July 13, 2023
```

Comment:
This should stay with `under development` since the exact date is TBA

Reply:
Fixed, thanks for letting me know